invenio_sipstore/models.py

```diff
@@ -175,20 +175,61 @@ def validate_key(self, filepath, filepath_):
     """Relation to the SIP along which given file was submitted."""
 
 
+class SIPMetadataType(db.Model):
+    """Type of the metadata added to an SIP.
+
+    The type describes the type of file along with an eventual schema used to
+    validate the structure of the content.
+    """
+
+    __tablename__ = 'sipstore_sipmetadatatype'
+
+    id = db.Column(db.Integer(), primary_key=True)
+    """ID of the SIPMetadataType object."""
+
+    name = db.Column(db.String(255), nullable=False, unique=True)
+    """The name of type of metadata (i.e. 'zenodo-json-1.0.0')."""
```

Comment:
we should probably make it zenodo-agnostic. ``...(i.e. `invenio-record-v1.0.0')``